components/text_recognition/browser/text_recognition_browser_export.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_TEXT_RECOGNITION_BROWSER_TEXT_RECOGNITION_BROWSER_EXPORT_H_
+#define BRAVE_COMPONENTS_TEXT_RECOGNITION_BROWSER_TEXT_RECOGNITION_BROWSER_EXPORT_H_
+
+#if defined(COMPONENT_BUILD)
+
+#if defined(WIN32)
+
+#if defined(TEXT_RECOGNITION_BROWSER_IMPLEMENTATION)
+#define TEXT_RECOGNITION_BROWSER_EXPORT __declspec(dllexport)
+#else
+#define TEXT_RECOGNITION_BROWSER_EXPORT __declspec(dllimport)
+#endif  // defined(TEXT_RECOGNITION_BROWSER_IMPLEMENTATION)
+
+#else  // defined(WIN32)
+
+#if defined(TEXT_RECOGNITION_BROWSER_IMPLEMENTATION)
+#define TEXT_RECOGNITION_BROWSER_EXPORT __attribute__((visibility("default")))
+#else
+#define TEXT_RECOGNITION_BROWSER_EXPORT
+#endif  // defined(TEXT_RECOGNITION_BROWSER_IMPLEMENTATION)
+
+#endif  // defined(WIN32)
+
+#else  // defined(COMPONENT_BUILD)
+
+#define TEXT_RECOGNITION_BROWSER_EXPORT
+
+#endif
```

Comment:
please use `base/component_export.h`.

and `COMPONENT_EXPORT(TEXT_RECOGNITION)` macro.

Reply:
Done.